app/helpers/passport.js

```diff
@@ -19,36 +22,87 @@ module.exports = function(passport) {
     passport.use(new GitHubStrategy({
         clientID: config.authentication.github.clientId,
         clientSecret: config.authentication.github.clientSecret,
-        callbackURL: '/login/callback'
```

Comment:
should this be hardcoded? if yes, and it's used more than once, can you pull it out and make it a "constant", in the sense of a CAPS_ONLY var at that top? that way we can change it all at once, easily, if need be